### ISSUE TITLE
The Solution Provider : xxxx is not authorized for getting the status of provided merchant : xxx

### ISSUE THREAD

**Issue by saraswat40** [amzn/amazon-pay-sdk-java#35]
We are trying to retrieve the status of a merchant account like so:

```
Config conf = new PayConfig()
                .withAccessKey(AMAZONPAYACCESSKEY)
                .withSecretKey(AMAZONPAYSECRETKEY)
                .withCurrencyCode(CurrencyCode.USD)
                .withSandboxMode(false)
                .withRegion(Region.US);

		Client client = new PayClient(conf);
		
		final GetMerchantAccountStatusRequest request = new GetMerchantAccountStatusRequest();
		request.setSellerId("XXXX");
		request.setMWSAuthToken("XXXXXXX");
		
		final GetMerchantAccountStatusResponseData resp = client.getMerchantAccountStatus(request);
```

Please let us know how to resolve this.

**Comment by bjguillot** [comment]
Try making your API call manually using the MWS scratchpad here: https://mws.amazonservices.com/scratchpad/index.html

"API Section" should be set to "Off-Amazon Payments"
"Operation" should be set to "GetMerchantAccountStatus".

If you get the same error message when using the scratchpad, that eliminates any potential issues with your code or the SDK code.

If you get the same issue with the scratchpad, and if you're certain all the keys are correct, it's possible that the auth token may have been revoked or has expired on the seller account.  To further troubleshoot that, you'll need to submit a new case to our team: https://sellercentral.amazon.com/cu/contact-us?ref_=py_contactus_shel_xx and select "Technical troubleshooting".

**Comment by saraswat40** [comment]
@bjguillot Yes, I am able to reproduce the issue on mws scratchpad as well. On the link you provided, I have another case open related to invalid ipn notifications since Mar 27, thats why I came here for help :)

**Comment by bjguillot** [comment]
@saraswat40 can you provide the Case # for your open case?

**Comment by saraswat40** [comment]
@bjguillot  6903997931. Thanks

**Comment by bjguillot** [comment]
OK, just sent an email to the email address used on the case.  Once I get your email reply, we can move troubleshooting for this issue as well as your IPN issue over to email and I'll close this issue.
